lib/identity/client.js

```diff
@@ -1450,6 +1450,25 @@ class Client extends PartialClient {
     const data = await response.json()
     return data.identities_credentials.map(IdentityMFADevices.decode)
   }
+
+  /**
+   * `removeMFADevice` removes an MFA device that is registered to an Identity
+   * @param {string} mfaID the ID of the MFA device
+   * @returns {object} No content if deleted or nonexistent
```

Comment:
return type should be boolean indicating success or failure, 204 and no content is what the api cares about, someone using the sdk just wants to know did it succeed or fail